src/native/libs/configure.cmake

```diff
@@ -1030,6 +1030,17 @@ check_include_files(
     GSS/GSS.h
     HAVE_GSSFW_HEADERS)
 
+if (HAVE_GSSFW_HEADERS)
```

Comment:
Can we move (cut & paste) this block https://github.com/dotnet/runtime/blob/3490f24ab7a8d609f6327f41f844e0a93370cadb/src/native/libs/System.Net.Security.Native/extra_libs.cmake#L3-L20 here instead? Would that work?

There is a CMake policy which allows includes across the directory includes that is used in coreclr/pal, I don't remmber if we are using it in native/libs; if that policy is effective here, we can delete the block, otherwise leave it as it is.

Reply:
I tagged you as a resident CMake expert. Honestly I have no idea what am I doing here and of I am doing it right...

I basically took that block and reduced it to minimum necessary to get the checks working. I'm not sure if moving it is possible since the extra_libs.cmake is also needed for the single-file host builds where everything is linked statically. Maybe including the extra_libs.cmake is an option? I didn't see that pattern anywhere though.

Reply:
Ah, I forgot about apphost path.